src/posts/official-ppa-20230206.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: "/news/official-ppa-20230206"
+date: "2023-02-06"
+title: "Announcing Apptainer Official PPA for Ubuntu"
+---
+
+We are pleased and excited to announce that Apptainer official PPA for Ubuntu 
+is now available!
+
+Rocky Linux users have long benefited from the availability of Apptainer on
+the EPEL repository. Apptainer, formerly known as Singularity, is widely used
+in the HPC community. Ubuntu is the one another major distribution for
+Apptainer users. Now there is great news for Ubuntu and Windows/WSL2/Ubuntu
+Apptainer users! Apptainer pre-built binaries are now available on the PPA
+for Ubuntu repository. Now you can install Apptainer using apt package
+manager. This makes it easy to install and update Apptainer on Ubuntu and
```

Comment:
Please remove the 3 sentences starting with "Apptainer," and going through the exclamation point.

Replace the next 2 sentences with this one: "Apptainer pre-built binaries are now also available in an Ubuntu PPA repository, installable with the apt package manager."

Reply:
Thank you, I'm preparing to update PR right now.